lldb/include/lldb/API/SBInstructionList.h

```diff
@@ -62,8 +66,7 @@ class LLDB_API SBInstructionList {
   friend class SBTarget;
 
   void SetDisassembler(const lldb::DisassemblerSP &opaque_sp);
-  bool GetDescription(lldb_private::Stream &description);
-
+  bool GetDescription(lldb_private::Stream &description, lldb::SBFrame *frame);
```

Comment:
Let's give this a default argument so you don't need to pass `nullptr` everywhere. 
```suggestion
  bool GetDescription(lldb_private::Stream &description, lldb::SBExecutionContext *exe_ctx = nullptr);
```

Reply:
:100: